packit/base_git.py

```diff
@@ -394,7 +398,7 @@ def push(self, refspec: str, remote_name: str = "origin", force: bool = False):
                 )
             ]
             if any(push_failed):
-                logger.debug(f"push_info flags: {pi.flags}")
+                logger.debug(f"The `push_info` flags: {pi.flags}")
```

Comment:
How about:

```suggestion
                logger.debug(f"Push flags: {pi.flags}")
```

to be consistent with "Push summary" above?